app/components/Nav/index.tsx

```diff
@@ -28,6 +29,12 @@ export const Nav = ({toc, categories}: NavProps) => {
             id="showArticles"
           />
           <ArticlesDropdown toc={toc} categories={categories || []} />
+          <MenuItem
+            primary={true}
+            link="https://chat.aisafety.info"
+            icon={<BotIcon />}
+            text="Stampy chatbot"
```

Comment:
This is the global nav right? Could we make it say "AI Safety chatbot" instead? :)